src/js/components/Drop/stories/Meter.js

```diff
@@ -38,5 +38,5 @@ SVGChild.parameters = {
 };
 
 export default {
-  title: 'Controls/Drop/SVG Child',
+  title: 'Controls/Drop/SVG child',
```

Comment:
Nice catch here. We will also need to add: `SVGChild.storyname = 'SVG child';`
Otherwise we get this nested behavior in storybook
<img width="216" alt="Screen Shot 2021-11-02 at 12 44 25 PM" src="https://user-images.githubusercontent.com/54560994/139925843-b3801305-9b70-406c-a24c-7121040cdbd1.png">